### PR TITLE
DE306: Fix issues with email validation - get rid of duplicate email element

### PR DESCRIPTION
**Pull request by IngageKrizJ**

### CODE REVIEW

File: crossroads.net/core/email_field/email_field_directive.js [hunk 1]
```diff
@@ -36,6 +36,12 @@ require('./email_field.html');
 
                 attachUniqueEmailFunctions(scope, emailElement, ngModel, $http, Session, User);
 
+                if(scope.focused && scope.focused == true) {
+                  $timeout(function() {
```
Comment (IngageBrewerM) [inline crossroads.net/core/email_field/email_field_directive.js]: Shouldn't there be a timeout value specified in this function?

Comment (IngageKrizJ) [inline crossroads.net/core/email_field/email_field_directive.js]: No, default is 0, which is what we want:
https://docs.angularjs.org/api/ng/service/$timeout

Comment (IngageBrewerM) [inline crossroads.net/core/email_field/email_field_directive.js]: Gotcha. Mind reviewing exactly why you need to do this with me in the morning? :confused:

Comment (IngageKrizJ) [inline crossroads.net/core/email_field/email_field_directive.js]: Yep, no problemo.

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**